### PR TITLE
NUTCH-1946: Added transitive dep to hbase-common-0.98.8-hadoop2.jar

### PR DESCRIPTION
**Pull request by jeroenvlek**
The comment says it all. I used the same exclusion pattern as with hbase-client, given my unfamiliarity with the project, someone else might want to skim over that.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -896,6 +896,38 @@
...
+      <dependency>
+        <groupId>org.apache.hbase</groupId>
+        <artifactId>hbase-common</artifactId>
+        <version>${hbase.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>avro</artifactId>
+          </exclusion>
+			<exclusion>
```
Comment (hsaputra) [inline pom.xml]: Style nit: the alignment of the XML tags are not right.

File: gora-hbase/pom.xml [hunk 2]
```diff
@@ -117,6 +117,18 @@
 
     <dependency>
       <groupId>org.apache.hbase</groupId>
+      <artifactId>hbase-common</artifactId>
+      <scope>compile</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-log4j12</artifactId>
```
Comment (hsaputra) [inline gora-hbase/pom.xml]: If we have exclude it in the main pom.xml do we need to exclude it again here?

File: pom.xml (hunk 1 above)
Comment (jeroenvlek) [inline pom.xml]: It was a direct copy of hbase-client, so I fixed the indentation of both.

File: gora-hbase/pom.xml (hunk 2 above)
Comment (jeroenvlek) [inline gora-hbase/pom.xml]: http://basementcoders.com/2010/05/maven-exclusions-not-always-inherited/

It depends. According to the link above, this exclusion actually causes the other exclusions in the main pom.xml to be *included* again. Is that on purpose? Do we then still need the other exclusions in the parent pom.xml?

My knowledge of Maven runs a bit short here. The difference with the example in the link above, is that  in the parent pom one exclusion is stated in a dependency management section, and the child pom just states its dependencies.

**Comment by lewismc** [comment]
@jeroenvlek can you please close off this pull request? We are in the process of addressing [GORA-443](https://issues.apache.org/jira/browse/GORA-443). Thank you

**Comment by purplewall1206** [comment]
hbase-0.94 didn't work well on my server , nutch will throw some exceptions like 

> java.lang.ClassNotFoundException: org.apache.gora.hbase.store.HBaseStore

so as the suggestion on [nutch wiki](http://wiki.apache.org/nutch/Nutch2Tutorial)
HBase 0.98.8-hadoop2 works well on my server(centOS 6)

**Pull request closed without merging**